test_cases/stdlib/check_pathlib.py

```diff
@@ -0,0 +1,7 @@
+# pyright: strict
```

Comment:
Which pyright error codes are missing from https://github.com/python/typeshed/blob/main/pyrightconfig.testcases.json? It looks to me like the pyright issues on your first commit were just because you had the type: ignore on the wrong line; not sure we need this: https://github.com/python/typeshed/actions/runs/6078220264/job/16489123042 

```suggestion
```

Reply:
Ah sorry, I just copied the file I was using to confirm that pyright used the same heuristic into the test case dir.